src/vm/exceptionhandling.cpp

```diff
@@ -5186,10 +5186,49 @@ BOOL IsSafeToHandleHardwareException(PCONTEXT contextRecord, PEXCEPTION_RECORD e
         IsIPInMarkedJitHelper(controlPc));
 }
 
+#ifdef _TARGET_ARM_
+static inline BOOL
```

Comment:
A nit - please move this to the same line with HandleArmSingleStep to match the surrounding code coding convention.